File: ZTP_Zadanie3/ZTP_Zadanie3/chain/wnioski/TransferRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace ZTP_Zadanie3.chain.wnioski
{
    class TransferRequest : Request
    {
        public TransferRequest(string title, DateTime date)
        {
            this.title = title;
            this.date = date;
        }
    }
}

File: ChainJava/src/chain/requests/TransferRequest.java
package chain.requests;

import java.util.Date;

/**
 * Created by Karolina_2 on 2015-04-01.
 */
public class TransferRequest extends Request {

    public TransferRequest(String title, Date date){
        this.title = title;
        this.date = date;
    }
}
